homeassistant/components/tuya/cover.py

```diff
@@ -272,7 +272,7 @@ def is_closed(self) -> bool | None:
             )
             is not None
         ):
-            return current_state in (True, "fully_close")
+            return current_state in (False, "fully_close")
```

Comment:
This does not match the PR description, for example, this also inverts it for other DPs, like `SITUATION_SET`.

Reply:
Ah, my bad, completely missed that 🤦‍♂️
Should I condition on `self.device.category`?

Reply:
Maybe we should add a lambda method to the entity descriptions instead.

Reply:
Please forgive my ignorance—I'm not following.
Isn't the entity descriptions just a dataclass? My understanding was that `self.device.status.get()` is what was actually getting the values.

Reply:
Yes, but if we extend the entity description (e.g. like we do in other platforms for Tuya as well), we could add, for example, a pre-processing method or something similar

Reply:
Okay, so to make sure I understand, I should add an `is_closed_fn` (name okay?) lambda method to the entity description?
If yes, should it be defined everywhere, or only for this category (for now)?
Sorry for driving you nuts, I'm just not that familiar with the nuts and bolts...

Reply:
>  I should add an is_closed_fn (name okay?)

Yeah, that might work, or just a `current_state_fn`?

Or even simpler, maybe add a `current_state_inverse` boolean to it? That would remove the need for a lambda

Reply:
And then condition on `current_state_inverse` in the `is_closed` method?

Reply:
Use that boolean to inverse the result of `return current_state in (False, "fully_close")`

E.g., `return inverse not is (current in (False, ...))`

Reply:
Thank you for all your help!